my-code/src/components/Movies.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import { FetchMovies } from "./FetchMovies";
+
+export function Movies() {
+
+    const [query, setQuery] = useState("");
+    const [disabled, setDisabled] = useState(true);
+    const [search, setSearch] = useState(false);
+
+    let apiKey = "798e7a67";
+
+    let min = 3;
+
+    function queryMovie() {
```

Comment:
Native methods are strongly not recommended to update or get DOM elements in React applications. As React uses `state` to trigger renders, this information it just got from DOM can be stale (from a previous render). If it is really necessary to interact with dom elements, take a look at this [section](https://reactjs.org/docs/refs-and-the-dom.html) in React documentation.

But for this case, React `state` will provide what you need to achieve the same result, even to get the value from input, [see more here](https://reactjs.org/docs/forms.html).